recipe/pypy-on-aarch-slowest.txt

```diff
@@ -0,0 +1,28 @@
+# Aggregated across several runs with different skips (because the full test suite timed out)
+# August 2021, scikit-learn v0.24.2, PyPy 7.3.5 on aarch64
+============================= slowest X durations =============================
+253.79s call     ensemble/tests/test_voting.py::test_gridsearch
+138.25s call     ensemble/tests/test_bagging.py::test_classification
+135.25s call     utils/tests/test_estimator_checks.py::test_check_estimator_clones
+106.52s call     tests/test_common.py::test_estimators[SequentialFeatureSelector(estimator=LogisticRegression(C=1))-check_estimator_sparse_data]
+101.77s call     ensemble/tests/test_common.py::test_ensemble_heterogeneous_estimators_behavior[stacking-classifier]
+95.01s call     ensemble/tests/test_common.py::test_ensemble_heterogeneous_estimators_behavior[stacking-regressor]
+76.04s call     feature_selection/tests/test_rfe.py::test_rfe_cv_groups
+72.23s call     tests/test_common.py::test_estimators[TSNE()-check_estimator_sparse_data]
+64.21s call     manifold/tests/test_t_sne.py::test_tsne_different_square_distances[True-manhattan-exact]
+64.08s call     manifold/tests/test_t_sne.py::test_tsne_different_square_distances[legacy-euclidean-exact]
+63.84s call     manifold/tests/test_t_sne.py::test_tsne_different_square_distances[True-euclidean-exact]
+63.24s call     manifold/tests/test_t_sne.py::test_tsne_different_square_distances[legacy-manhattan-exact]
+61.57s call     linear_model/tests/test_coordinate_descent.py::test_linear_models_cv_fit_for_all_backends[MultiTaskLassoCV-loky]
+59.26s call     ensemble/tests/test_bagging.py::test_regression
+52.94s call     ensemble/tests/test_forest.py::test_class_weights[RandomForestClassifier]
+52.75s call     feature_selection/tests/test_sequential.py::test_nan_support
+52.38s call     neighbors/tests/test_kde.py::test_kde_sample_weights
+47.97s call     manifold/tests/test_t_sne.py::test_uniform_grid[exact]
+47.28s call     utils/tests/test_estimator_checks.py::test_check_estimator
+45.71s call     ensemble/tests/test_bagging.py::test_parallel_classification
+44.37s call     decomposition/tests/test_dict_learning.py::test_sparse_coder_parallel_mmap
+42.23s call     manifold/tests/test_t_sne.py::test_sparse_precomputed_distance
+40.57s call     neural_network/tests/test_mlp.py::test_n_iter_no_change_inf
+40.38s call     utils/tests/test_random.py::test_sample_without_replacement_algorithms
+39.31s call     tests/test_common.py::test_estimators[RFECV(estimator=LogisticRegression(C=1))-check_estimator_sparse_data]
```

Comment:
How do these times compare to CPython? Is there an obvious common factor between the slowdowns?

Reply:
Answered in [this](https://github.com/conda-forge/scikit-learn-feedstock/pull/159#issuecomment-896550966) comment.